modules/administration_manual/pages/appliance/wnd_setup.adoc

```diff
@@ -0,0 +1,65 @@
+= Windows Network Drive Configuration
```

Comment:
As you're using WND in the first paragraph, please add it as (WND) at the end of the title.